src/components/flame-graph/FlameGraph.js

```diff
@@ -116,7 +171,12 @@ class FlameGraph extends React.PureComponent<Props> {
         break;
       }
       case 'ArrowUp': {
-        const [callNodeIndex] = callTree.getChildren(selectedCallNodeIndex);
+        const children = callTree.getChildren(selectedCallNodeIndex);
+        // The order of the children returned from getChildren does
+        // not correspond to the paint order in the flame graph, so
+        // the one chosen for selection might look rather arbitrary.
```

Comment:
Actually I don't think this is so arbitrary: I believe the children are sorted by running time, so it actually looks quite logical to me.
This also means that the first children will be the widest of all children, so you can just test it, instead of using `find`, to know if it's possible to move up.

Reply:
Yes you're right. I was thinking about when the user navigates with the keyboard. If he or she doesn't realize that it is the largest box that is selected when going up, it might be confusing and look arbitrary :-) Even I didn't realize that, so thanks for the tip for just testing the first candidate :+1: 

But I removed the comment, might not be of so much value mentioning it there.